defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+app_path: /home/ubuntu/test
```

Comment:
Out of curiosity, why did you elect to put `app_path` at the top level instead of `capistrano.app_path`?